packages/publications/src/Commands/Helpers/InputStreamHandler.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyde\Publications\Commands\Helpers;
+
+use function array_shift;
+use function explode;
+use function fgets;
+use Hyde\Hyde;
+use function trim;
+
+/**
+ * Collects an array of lines from the standard input stream. Feed is terminated by a blank line.
+ *
+ * @internal
+ *
+ * @see \Hyde\Framework\Testing\Unit\InputStreamHandlerTest
+ */
+class InputStreamHandler
```

Comment:
Change termination sequence from empty line (as input might have empty lines) to something like `<<<` (and possibly also supporting an EOT control character)

Reply:
Resolved in https://github.com/hydephp/develop/pull/936 and https://github.com/hydephp/develop/pull/937